composer.json

```diff
@@ -24,6 +23,7 @@
         "ext-xhprof": "You need to install either xhprof or uprofiler to use XHGui.",
         "ext-uprofiler": "You need to install either xhprof or uprofiler to use XHGui.",
         "ext-mongo": "Mongo is needed to store profiler results for PHP < 7.",
-        "ext-mongodb": "Mongo is needed to store profiler results for PHP > 7."
+        "ext-mongodb": "Mongo is needed to store profiler results for PHP > 7.",
+        "alcaeus/mongo-php-adapter": "Mongo PHP Adapter is required for PHP >7 (when using ext-mongodb)"
```

Comment:
Could the `install.php` script be updated to always install the mongodb package? That way people won't have to do more when installing xhgui.